spec/aerospike/query_spec.rb

```diff
@@ -87,7 +87,7 @@
         count = 0
         rs.each do |rec|
           count += 1
-          expect(rec.bins.keys).to contain_exactly("bin1", "bin2", "bin3", "bin4")
+          expect(rec.bins.keys).to contain_exactly("bin1", "bin2", "bin3", "bin4", "bin5")
```

Comment:
Why is this test affected by a test coming after?

Reply:
Have reverted the changes!